node-launchpad/src/components/status.rs

```diff
@@ -798,19 +813,54 @@ impl Component for Status<'_> {
         ]);
 
         let connection_mode_string = match self.connection_mode {
-            ConnectionMode::HomeNetwork => "Home Network",
-            ConnectionMode::UPnP => "UPnP",
-            ConnectionMode::CustomPorts => &format!(
+            ConnectionMode::HomeNetwork => "Home Network".to_string(),
```

Comment:
So are we still presenting it as 'home network' to the user? Or should this also be renamed to 'relay'?